mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -87,6 +87,10 @@
          so that the user doesn't forget about the website still being open. This text is shown in
          the notification. Clicking the notification will have the same effect as clicking the "erase"
          button in the application. -->
+
+    <string name="notification_erase_title_android_14">Erase browsing history?</string>
```

Comment:
Nit: the comment above refers to the `notification_erase_text` string (comment mentions "Clicking the notification will have the same effect as clicking the "erase" button in the application", which is not what happens if you click "Erase browsing history?").

Reply:
I left the comment above the title and text strings because it refers to all of them. Do you think it would be better to duplicate the comment? ( or even leave it also for the previous `notification_erase_text`?

Reply:
@mcarare On Pontoon, localizers typically see only one string at a time, with their own unique comment attached. The only "issue" I saw in this comment, is that it says that "Clicking the notification will have the same effect as clicking the "erase" button in the application" - which I *think* is not what happens if you click "Erase browsing history?", which is the string directly attached to that comment.

The `notification_erase_text `string is pretty clear so I don't think it needs a comment in this specific case.

Reply:
I did not add that comment, this was the comment for the `notification_erase_text` and it's as correct for the android 14 string as it was for the previous one. Clicking the notification will erase browsing sessions.

As a side note, I am not sure why the comment was needed in the first place, it does not seem to me like it adds context for the translators.